src/features/farming/crops/components/Soil.tsx

```diff
@@ -47,6 +48,7 @@ export const Soil: React.FC<Props> = ({
   plantedCrop,
   className,
   showCropDetails,
+  isRemoving,
```

Comment:
Not a blocker, but we are beginning to leak the implementation details here.

In this case, the component shouldn't be aware of it's use case (i.e. is removing a crop) - we want UI components to remain decoupled and 'dumb'

In these cases I would suggest just adding the negative progress bar image directly in the Field.tsx component. After 2-3 uses we can then abstract it into a separate file.